src/components/GameView/GameOverDialog.vue

```diff
@@ -90,6 +124,27 @@ export default {
 
       return 'loss-img';
     },
+    currentMatch(){
+      return this.game.currentMatch
+    },
+    matchGameStats(){
+      if(!this.game.currentMatch?.games) return []
+      const result= this.game.currentMatch.games.map(game=>{
+        if (game.result === -1) return 'I'
+        if(game.result === 0 || game.result === 1){
+          if(game.p0 === this.player.id) return 'W'
+          return 'L'
+        }
+        return 'D'
+      })
+      return result
```

Comment:
You're declaring a variable just to return it, you could directly return the result without needing to assign it at all.
```
return this.game.currentMatch?.games?.map?.((game) => { ... });
```

Reply:
I think previously I was console logging the result to see if it was the desired shape. Declaring a variable made it easier